dev/Repeater/APITests/FlowLayoutCollectionChangeTests.cs

```diff
@@ -306,6 +306,78 @@ public void CanReplaceSingleItem()
             });
         }
 
+        [TestMethod]
+        public void CanMoveItem()
+        {
+            CustomItemsSource dataSource = null;
+            RunOnUIThread.Execute(() => dataSource = new CustomItemsSource(Enumerable.Range(0, 10).ToList()));
+            ScrollViewer scrollViewer = null;
+            ItemsRepeater repeater = null;
+            var viewChangedEvent = new ManualResetEvent(false);
+            int elementsCleared = 0;
+            int elementsPrepared = 0;
+            int elementsIndexChanged = 0;
+
+            RunOnUIThread.Execute(() =>
+            {
+                repeater = SetupRepeater(dataSource, ref scrollViewer);
+                scrollViewer.ViewChanged += (sender, args) =>
+                {
+                    if (!args.IsIntermediate)
+                    {
+                        viewChangedEvent.Set();
+                    }
+                };
+
+                repeater.ElementPrepared += (sender, args) => { elementsPrepared++; };
+                repeater.ElementIndexChanged += (sender, args) => { elementsIndexChanged++; };
+                repeater.ElementClearing += (sender, args) => { elementsCleared++; };
+
+                scrollViewer.ChangeView(null, 300, null, true);
+            });
+
+            Verify.IsTrue(viewChangedEvent.WaitOne(DefaultWaitTime), "Waiting for ViewChanged.");
+            IdleSynchronizer.Wait();
+
+            RunOnUIThread.Execute(() =>
+            {
+                var realized = VerifyRealizedRange(repeater, dataSource);
+                Verify.AreEqual(4, realized);
+
+                Log.Comment("Move before realized range.");
+                dataSource.Move(oldIndex: 0, newIndex: 1, reset: false);
+                repeater.UpdateLayout();
+
+                realized = VerifyRealizedRange(repeater, dataSource);
+                Verify.AreEqual(4, realized);
+
+                Log.Comment("Replace in realized range.");
```

Comment:
What are we replacing here? Looking at the move operation below, it seems that we are just moving items inside the realized range.

Reply:
My bad, I forgot to update all the comments

Reply:
Happens :) Thanks for fixing those so quickly!